Tests/Resources/App/AppKernel.php

```diff
@@ -47,7 +47,7 @@ protected function prepareContainer(ContainerBuilder $container)
         $container->setParameter('cmf_testing.bundle_fqn', 'DTL\Bundle\PhpcrMigrations\PhpcrMigrationsBundle');
     }
 
-    protected function getKernelParameters()
+    protected function getKernelParameters(): array
```

Comment:
This is required for Symfony 6 and requires us to drop support for PHP 5.6 else we would have needed to create a compatibility layer, which I think is not worth the work for php 5.6